Index.html

```diff
@@ -0,0 +1,153 @@
+          <hr class="line">
+        </div>
+        
+        <div class="shedule-container">
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/dj.png"  alt=" ">
+            <h3>Openning with JD</h3>
+            <p class="text1">Listen to theamazing talented DJs in Bemanda. Give you what they do best in their career</p>
+          </div>
+
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/artist.png" alt=" ">
+            <h3>1<sup>st</sup> Artist</h3>
+            <p>The fist artist will take over the stage after the Djs have finish performing their talent</p>
+          </div>
+
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/artist.png" alt=" ">
+            <h3>2<sup>nd</sup> Artist</h3>
+            <p>The second artist will take over the stage after the 1st artist have finish performing their talent</p>
+          </div>
+
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/artist.png" alt=" ">
+            <h3>3<sup>th</sup> Artist</h3>
+            <p>The third artist will take over the stage after the 2nd artist  have finish performing their talent</p>
+          </div>
+
+          <div  class="shedule-cards">
+            <img class="image image1" src="Images/Musician/main.png" alt=" ">
+            <h3>Main Artist</h3>
+            <p>The main artist of the day will close the program and trust US he will blow your mind</p>
+          </div>
+        </div>
+        <div class="button-container">
+        <button class="ticket-button" type="submit">
+             Grab a ticket today
+        </button>
+        </div>
+      </section>
+
+      <section class="artist-section setoff">
+        <div class="text-time">
+        <h3 class="text-header2">Artists</h3>
+        <hr class="line2">
+        </div>
+
+        <div class="artist-container">
+       
+        </div>
+
+
+      </section>
+      <div class="seemore">
+        <span id="span">see more artist &nabla;</span>
+      </div>
+
+      <section class="spnosor-section">
+        <div class="text-time">
+             <h3  class="text-header">Spnosors</h3>
+             <hr class="line">
+        </div>
+        <div class="spnosor-container">
+          <img class="spnosor-imgages" src="Images/Musician/MTN_Logo.svg.png" alt=" ">
+          <img class="spnosor-imgages" src="Images/Musician/58429064a6515b1e0ad75aba.png" alt=" ">
+          <img class="spnosor-imgages" src="Images/Musician/airtel-logo-vector-removebg-preview.png" alt=" ">
+          <img class="spnosor-imgages" src="Images/Musician/camtel.jpg" alt=" ">
+          <img class="spnosor-imgages" src="Images/Musician/Glo_button-removebg-preview.png" alt=" ">
+        </div>
+      </section>
+
+      <footer>
+        <div>
+          <a><img class="footer-image" src="Images/logo_image/logo-image.png"></a>
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

Reply:
Kindly, I suggest that when using any "A" tag, indicate the href attribute, if it doesn't contain a link, use "#".